files/en-us/web/api/keyboardevent/code/code_values/index.html

```diff
@@ -1713,16 +1629,16 @@ <h3 id="Code_values_on_Mac">Code values on Mac</h3>
  </tbody>
 </table>
 
-<h3 id="Code_values_on_Linux_X11_When_scancode_is_available">Code values on Linux (X11) (When scancode is available)</h3>
+<h2 id="Code_values_on_Linux_X11_When_scancode_is_available">Code values on Linux (X11) (When scancode is available)</h2>
```

Comment:
I think it would be better to omit the paren here:

```suggestion
<h2 id="Code_values_on_Linux_X11">Code values on Linux (X11)</h2>
```